### PR TITLE
navigation implementation

### PR DESCRIPTION
**Pull request by bartekkleczar**
- navigation code

- implemented navigation in HomeScreen

- added StudentsScreen with implemented navigation

### CODE REVIEW

File: app/src/main/java/com/example/timecontrol/navigation/Screen.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.example.timecontrol.navigation
+
+sealed class Screen(val route: String){
+    object HomeScreen: Screen("home_screen")
+    object StudentsScreen: Screen("Students_screen")
```
Comment (ReptilianEye) [inline app/src/main/java/com/example/timecontrol/navigation/Screen.kt:5]: Rename route names to have the same case for example snake_case

File: app/src/main/java/com/example/timecontrol/navigation/Navigation.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        //Students Screen
+        composable(route = Screen.StudentsScreen.route) {
+            RootLayout(navController = navController,
+                localization = 0,
+                content = {
+                StudentsScreen()
+            })
+        }
```
Comment (ReptilianEye) [inline app/src/main/java/com/example/timecontrol/navigation/Navigation.kt:41]: One day there will be ScheduleScreen, right?

**Review by ReptilianEye — CHANGES_REQUESTED** [review]
Well done, I would rename route names and it would be perfect

File: app/src/main/java/com/example/timecontrol/navigation/Navigation.kt (hunk 2 above)
Comment (bartekkleczar) [reply app/src/main/java/com/example/timecontrol/navigation/Navigation.kt:41]: I forgot, I'll implement it for future

File: app/src/main/java/com/example/timecontrol/navigation/Screen.kt (hunk 1 above)
Comment (bartekkleczar) [reply app/src/main/java/com/example/timecontrol/navigation/Screen.kt:5]: done

File: app/src/main/java/com/example/timecontrol/navigation/Navigation.kt [hunk 3]
```diff
@@ -20,24 +21,33 @@ fun Navigation(viewModel: DatabaseViewModel, context: Context, quote: Quote) {
...
                 localization = 0,
                 content = {
-                StudentsScreen()
-            })
+                    StudentsScreen()
+                })
+        }
+
+        //Schedule Screen
+        composable(route = Screen.StudentsScreen.route) {
```
Comment (ReptilianEye) [inline app/src/main/java/com/example/timecontrol/navigation/Navigation.kt:45]: I guess you wanted to write Screen.ScheduleScreen.route

File: et --hard HEAD~1
Comment (ReptilianEye) [inline et --hard HEAD~1:1]: what is this file?

**Review by ReptilianEye — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/timecontrol/navigation/Navigation.kt (hunk 3 above)
Comment (bartekkleczar) [reply app/src/main/java/com/example/timecontrol/navigation/Navigation.kt]: Replaced

**Review by ReptilianEye — APPROVED** [review]
Well done!

**Pull request merged**